### DISCUSSION TITLE
Can't install Copilot on VSCode on linux

### DISCUSSION THREAD

**Discussion by Overloader6** [community/community#14372] [Copilot Conversations] [answered]
It tells me I need version 1.57+ of vscode. According to `code --version` I have 1.66 installed
![image](https://user-images.githubusercontent.com/22007229/161931228-6401d2c0-5ba9-4541-ad3e-408a1c61f1bf.png)

**Comment by davecheney** [comment, marked as answer]
Most linux distributions bundle an older version of VSCodium, this does not work with copilot. You must use VSCode.
